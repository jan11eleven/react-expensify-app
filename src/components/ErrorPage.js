import React from "react";
import { Link } from "react-router-dom";

const ErrorPage = () => (
  <div>
    Error 404
    <Link to="/">Back to page</Link>
  </div>
);

export default ErrorPage;
