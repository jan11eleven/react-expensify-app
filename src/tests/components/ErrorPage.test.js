import React from "react";
import { shallow } from "enzyme";
import ErrorPage from "../../components/ErrorPage";

test("should render Error Page correctly", () => {
  const wrapper = shallow(<ErrorPage />);

  expect(wrapper).toMatchSnapshot();
});
