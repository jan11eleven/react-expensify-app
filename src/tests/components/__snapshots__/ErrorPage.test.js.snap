// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Error Page correctly 1`] = `
<div>
  Error 404
  <[object Object]
    to="/"
  >
    Back to page
  </[object Object]>
</div>
`;
